torchvision/prototype/features/_feature.py

```diff
@@ -97,6 +97,10 @@ def _make_repr(self, **kwargs: Any) -> str:
 
     @property
     def _F(self) -> ModuleType:
+        # This implements a lazy import of the functional to get around the cyclic import. This import is deferred
+        # until the first time we need reference to the functional module and it's shared across all instances of
+        # the class. The deferred lazy import avoids the DataLoader issue described at
+        # https://github.com/pytorch/vision/pull/6219#discussion_r912721945
```

Comment:
I'm kind of anxious about this since we don't have a repro. @vfdev-5 could you try to replicate the error you saw? All our current test indicate that the lazy loading indeed works with the DataLoader, but maybe we also have a gap in the tests.

Reply:
You are right to be anxious. I did reproduce the issue first to ensure I'm not breaking anything. See commit 692d6b9459f14092771c8229531a93a0a6ad6f6a which restored an earlier approach that @vfdev-5 and I came up with and confirmed that it was still failing on my setup. I've also ran the new tests that you proposed (and rolled back) and all passed with exception to the unrelated voc issue. This doesn't actually guarantees we don't have a gap on the tests, but at least we covered the basics. 

I'm happy to do more testing to make you more comfortable with this. We can also revert this PR and reinstate the runtime imports if it turns out to be causing issues. Thoughts?

Reply:
>  I did reproduce the issue first to ensure I'm not breaking anything. 

Ah, I didn't know that. In that case I'm fine now, especially since we can always revert.

Reply:
Maybe we can link this discussion instead?